jaeger-core/src/main/java/io/jaegertracing/internal/JaegerObjectFactory.java

```diff
@@ -0,0 +1,46 @@
+package io.jaegertracing.internal;
+
+import io.jaegertracing.internal.clock.Clock;
+import io.jaegertracing.internal.metrics.Metrics;
+import io.jaegertracing.spi.BaggageRestrictionManager;
+import io.jaegertracing.spi.Reporter;
+import io.jaegertracing.spi.Sampler;
+import io.opentracing.ScopeManager;
+
+import java.util.List;
+import java.util.Map;
+
+public class JaegerObjectFactory {
```

Comment:
What is a purpose of this class? Can you please document it here?

Reply:
I will very soon.